test.disabled/ejabberd_tests/tests/vcard_SUITE.erl

```diff
@@ -144,6 +151,15 @@ init_per_group(ldap_only, Config) ->
         _ ->
             {skip, "this group is only for ldap vCard backend"}
     end;
+init_per_group(ro_rsm, Config) ->
+    VCardConfig = ?config(mod_vcard, Config),
+    case proplists:get_value(backend, VCardConfig) of
+        Backend when Backend =:= undefined;
+                     Backend =:= mnesia ->
+            Config;
+        _ ->
+            {skip, "RSM support is only implemented for the mnesia backend"}
```

Comment:
Right, it seams the changes are backend-agnostic if so tests need to pass will all possible backends. After merging a change to master we will maintain it. To maintain it properly we need tests covering all backends.

I'm OK with only `mnesia` backend as long as the implementation returns error when RSM tried with other backend.